recipes/distmetrics/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "distmetrics" %}
+{% set version = "0.0.2" %}
+{% set python_min = "3.12" %} 
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/distmetrics-{{ version }}.tar.gz
+  sha256: c3e57c21616fa944e7014e1c5dadcb04d324e17f0638d97b86b87669f74b4743
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+    - affine
+    - rasterio
+    - numpy
+    - scikit-image
+    - matplotlib-base
+    - scipy
+    - pydantic
+    - astropy-base
+    - einops
+    - backoff
+    - asf_search
+    - shapely
+    - geopandas
+    - pytorch
+    - pydantic
+    - asf_search
+    - tqdm
+    - dill
+    - mpire
+    - pandas
+
+test:
+  imports:
+    - distmetrics
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/cameronmarshak/distmetrics
```

Comment:
This is giving me a 404. Can you fix it in the feedstock?

Reply:
Sorry about that. Thank you, @ocefpaf!